nextjs/packages/next/next-server/server/next-server.ts

```diff
@@ -246,6 +247,10 @@ export default class Server {
     this.router = new Router(this.generateRoutes())
     this.setAssetPrefix(assetPrefix)
 
+    if (experimental.initServer) {
+      experimental.initServer()
```

Comment:
Can we make this `await  experimental.initServer()`?

Reply:
Use case: `connectToMongoDB()` for those who don't use custom server but are not running serverless